domain-models-runtime/src/main/java/org/folio/rest/tools/utils/ValidationHelper.java

```diff
@@ -41,85 +36,77 @@ public static Errors createValidationErrorMessage(String field, String value, St
     return e;
   }
 
-  public static void handleError(Throwable error, Handler<AsyncResult<Response>> asyncResultHandler){
-
-    if(error != null){
+  public static void handleError(Throwable error, Handler<AsyncResult<Response>> asyncResultHandler) {
+    if (error != null) {
       Response r = null;
-      Map<Character,String> errorMessageMap = PgExceptionUtil.getBadRequestFields(error);
-      if(errorMessageMap != null){
+      Map<Character, String> errorMessageMap = PgExceptionUtil.getBadRequestFields(error);
+      if (errorMessageMap != null) {
         //db error
         String desc = errorMessageMap.getOrDefault('D', "");
         String mess = errorMessageMap.getOrDefault('M', "");
-        try{
-          if(isInvalidUUID(mess)){
+        try {
+          if (isInvalidUUID(mess)) {
             int start = mess.indexOf('\"');
-            int end = mess.indexOf('\"', start+1);
-            String value = "";
-            if(start != -1 && end != -1 && start < end){
-              value = mess.substring(start+1, end);
+            int end = mess.indexOf('\"', start + 1);
+            String UUIDvalue = "";
```

Comment:
Can you rename it to `uuidValue` to comply with
* https://dev.folio.org/guidelines/contributing/#style-conventions
* https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
* https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-nameshttps://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

Reply:
String UUIDvalue updated to uuidValue .